modules/ROOT/pages/7.8.0-release-notes.adoc

```diff
@@ -69,6 +69,52 @@ The {productname} {release-version} release includes an accompanying release of
 
 For information on the **<Premium plugin name 1>** plugin, see: xref:<plugincode>.adoc[<Premium plugin name 1>].
 
+=== Math
+
+The {productname} {release-version} release includes an accompanying release of the **Math** premium plugin.
+
+**Math** includes the following addition.
+
+==== New `extended_mathml_attributes` and `extended_mathml_elements` options.
+// #TINY-11756
+
+To improve flexibility when working with MathML content, especially in cases where new attributes or elements are not yet supported by DOMPurify, two new configuration options have been introduced: xref:math.adoc#extended-mathml-elements[extended_mathml_elements] and xref:math.adoc#extended-mathml-attributes[extended_mathml_attributes].
+
+Prior to {release-version}, MathML elements and attributes could be configured informally, but were not officially supported. With enhanced security measures introduced in {productname} {release-version}, MathML content is now filtered separately from HTML using DOMPurify, and any unsupported elements or attributes are stripped from the editor content. This change increased security but removed the ability to allow certain MathML-specific content.
```

Comment:
Not quite sure this is correct, might have described it poorly myself. Previously it did treat the elements the same as everything else with no consideration for the mathml-stuff. With this it specifically allows some elements inside of a <math>-element ( but not outside of it ). Same for attributes.

Reply:
@HAFRMO No problem, can you review the suggested change based on your feedback?
```suggestion
Prior to {release-version}, MathML elements and attributes were treated the same as all other content, with no special handling for MathML-specific structures. As a result, unsupported elements and attributes were either retained without validation or stripped without regard to their MathML context.

In {productname} {release-version}, MathML content is now filtered separately from general HTML using DOMPurify. As part of this change, support has been added to selectively allow specific elements and attributes *within* a `<math>` element using the new configuration options.
```